### PR TITLE
vector streams

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Vektor Chunks Stream nach Python](https://trello.com/c/OvgxPSsg/1194-vektor-chunks-stream-nach-python)

File: geoengine/workflow.py [hunk 1]
```diff
@@ -744,6 +745,165 @@ def merge_tiles(tiles: List[xr.DataArray]) -> Optional[xr.DataArray]:
...
+            geometry = gpd.GeoSeries.from_wkt(data_frame[api.GEOMETRY_COLUMN_NAME])
+            del data_frame[api.GEOMETRY_COLUMN_NAME]  # delete the duplicated column
+
+            geo_data_frame = gpd.GeoDataFrame(
+                data_frame,
+                geometry=geometry,
+                crs=spatial_reference,
+            )
+
+            # TODO: separate timestamps into start and end
```
Comment (jdroenner) [inline geoengine/workflow.py:775]: could we write them as two columns?

File: tests/test_workflow_vector_stream.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
+'''Tests for WMS calls'''
```
Comment (jdroenner) [inline tests/test_workflow_vector_stream.py:1]: wrong title?

Comment (ChristianBeilschmidt) [reply tests/test_workflow_vector_stream.py]: [150b749](https://github.com/geo-engine/geoengine-python/pull/112/commits/150b7498eb31b953f2e9c30e139917a4cf81d762)

**Comment by jdroenner** [comment]
bors merge

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**